c++/src/capnp/layout.c++

```diff
@@ -1131,7 +1131,7 @@ struct WireHelpers {
     word* oldPtr = followFars(oldRef, refTarget, oldSegment);
 
     KJ_REQUIRE(oldRef->kind() == WirePointer::STRUCT,
-        "Message contains non-struct pointer where struct pointer was expected.") {
+        "Schema mismatch: Message contains non-struct pointer where struct pointer was expected.") {
```

Comment:
Aren't there some other errors in this file that should be marked as "schema mismatch"? I'd expect to see e.g. the case where a list is expected but a non-list pointer is found.